### PR TITLE
Use `getRootNode()` instead of `root()` as it got deprecated since Symfony 4.2

### PR DESCRIPTION
**Pull request by simPod**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
 	"require": {
 		"php": "~7.2",
-		"symfony/config": "~4.0",
-		"symfony/console": "~4.0",
-		"symfony/dependency-injection": "~4.0",
-		"symfony/http-kernel": "~4.0",
-		"symfony/twig-bundle": "~4.0",
-		"symfony/yaml": "~4.0",
+		"symfony/config": "^4.4|^5.0",
+		"symfony/console": "4.4|^5.0",
```
Comment (mhujer) [inline composer.json]: shouldn't it be `^4.4`?

**Comment by simPod** [comment]
Sure. I'm just spamming force pushes here. Did not want to bother with old symfony versions, had no idea someone might see it. Thanks!

File: composer.json (hunk 1 above)
Comment (simPod) [reply composer.json]: Sure. I'm just spamming force pushes here. Did not want to bother with old symfony versions, had no idea someone might see it. Thanks!

**Comment by soukicz** [comment]
Is this just waiting for merge or is there some other compatibility issue? We are just upgrading to Symfony 5 and this is blocking the upgrade. I am asking to know if we should try to solve compatibility and send PR or if this is ready and we can just try to test it.

**Comment by insekticid** [comment]
@soukicz 

composer.json

```yaml
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/simPod/Tracy-Blue-Screen-Bundle"
        }
    ],
    "require": {
        "vasek-purchart/tracy-blue-screen-bundle": "dev-symfony5",

....
```

**Pull request closed without merging**